integration/network_partition_test.go

```diff
@@ -86,6 +86,10 @@ func TestNetworkPartition5MembersLeaderInMajority(t *testing.T) {
 
 	// recover network partition (bi-directional)
 	recoverPartition(t, majorityMembers, minorityMembers)
+
+	// wait extra election timeout
+	time.Sleep(2 * clus.Members[0].electionTimeout())
```

Comment:
does this apply to the other partition tests too? I've also seen failures on the 4 member test.